.travis.yml

```diff
@@ -6,6 +6,10 @@ notifications:
 sudo: false
 script:
 - npm test
+before_install:
+- openssl aes-256-cbc -K $encrypted_ca4a154c2e6b_key -iv $encrypted_ca4a154c2e6b_iv -in config/tenon.json.enc -out config/tenon.json -d
+before_deploy:
+- test $TRAVIS_TEST_RESULT = 0
```

Comment:
If we do this does this mean the build cant fail? https://github.com/travis-ci/docs-travis-ci-com/commit/e8987d91d15c05a6cbd48ab5eb699ae3ee1724a3#diff-af00654bc38c4015862dffd1e60ff7e5R178